src/topology/algebra/infinite_sum.lean

```diff
@@ -1198,6 +1198,7 @@ begin
 end
 
 local attribute [instance] topological_add_group.regular_space
+local attribute [-instance] regular_space.to_t0_space
```

Comment:
Why is this needed?

Reply:
Because with `local attribute [instance] topological_add_group.regular_space` there is a loop, as we only need t0 space for a regular space now, and regular implies t0, I guess this is why `topological_add_group.regular_space` isn't a global instance to begin with.

Reply:
Could you add a comment above those two lines to explain this?

Reply:
I'll add the comment when I have a minute later :)

Reply:
Which lemmas actually need this? Can you wrap a `section` around the ones that do? Or maybe even just use `letI` in the proof?

Reply:
All the lemmas in the section uniform group after the local instances need it, I added another section to make it clearer, but the new section does nothing right now.